priv/code_navigation/elvis.config

```diff
@@ -0,0 +1,30 @@
+[{ elvis
+ , [
+    { config,
+      [ #{ dirs    => ["src"]
+         , filter  => "*.erl"
+         , ruleset => erl_files
+         , rules   => [ {elvis_style, invalid_dynamic_call, #{ignore => [ els_compiler_diagnostics
```

Comment:
I guess the ignore rules come from the main `elvis.config`. These do not really make sense in the `priv` sample config, so I would remove them.

Reply:
Yes, I just copied the main config over.